### DISCUSSION TITLE
About plots

### DISCUSSION THREAD

**Discussion by Bitaro** [Chia-Network/chia-blockchain#4789] [Q&A]
Hello guys
I'm starting rigth now and i'm count 9 plots here already.
I start making this plots at my work pc where i start Chia for the first time.
If i like to start a new Chia environment at a new pc (probably at home) with new keys and mnemonics and wallet, and forget the first start, it's possible "import" all the 9 plots already maded at work to the new without problem? 
Or the 9 plots who i'm with now are only exclusivley for the first Chia environment i start?
Thanks in advance
Best regards
Vitor

**Comment by lvcivs** [comment]
The plots are bound to your keys. So if you start over with new keys at home, you cannot use your previous plots.
